engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/content/views/content-host-errata.html

```diff
@@ -1,3 +1,11 @@
+<form id="errataActionForm" method="post" action="/katello/remote_execution">
```

Comment:
I don't think you meant to duplicate this form

Reply:
Yeah, we are currently trying to remember what this was supposed to fix ;-)

I will remove the added duplicate for now and try to do some tests.